pom.xml

```diff
@@ -157,32 +157,32 @@
             <dependency>
                 <groupId>com.fasterxml.jackson.dataformat</groupId>
                 <artifactId>jackson-dataformat-yaml</artifactId>
-                <version>2.9.0</version>
```

Comment:
I think we kept these hardcoded versions because @dflemstr said variable versions don't play well with maven tools that upgrade dependency versions. I don't have strong feelings though. And looking at the [versions-maven-plugin](http://www.mojohaus.org/versions-maven-plugin/examples/update-properties.html), I think it can handle variable versions.

@dflemstr can you explain again why you didn't like this? I'm just curious.

Reply:
I think this causes problems if something depending on helios as a dependency also specifies a `jackson.version` property. But then again, we should be consistent in this pom and either use properties for repeated versions like this or no use it at all.

Reply:
Looks like we don't really use properties for repeated versions.